consensus/polybft/state_transaction.go

```diff
@@ -246,45 +263,23 @@ func (cm *CommitmentMessage) ContainsStateSync(stateSyncIndex uint64) bool {
 
 // BundlesCount calculates bundles count contained in given CommitmentMessge
 func (cm *CommitmentMessage) BundlesCount() uint64 {
-	return (cm.ToIndex - cm.FromIndex + cm.BundleSize) / cm.BundleSize
+	return uint64(math.Ceil((float64(cm.ToIndex) - float64(cm.FromIndex) + 1) / float64(cm.BundleSize)))
```

Comment:
I am not sure if original code is so hard to understand that we need to change it? It just simple integer divide arithmetics

Reply:
Yeah, this can be changed to the original.

Reply:
Fixed [43050c7](https://github.com/0xPolygon/polygon-edge/pull/796/commits/43050c73e6a58bdc050ef8e7e0734e30b81072f8)